src/containers/ListingPage/ListingPage.shared.js

```diff
@@ -173,66 +193,71 @@ export const handleSubmit = parameters => values => {
   const listingId = new UUID(params.id);
   const listing = getListing(listingId);
 
-  const {
-    bookingDates,
-    bookingStartTime,
-    bookingEndTime,
-    bookingStartDate, // not relevant (omit)
-    bookingEndDate, // not relevant (omit)
-    quantity: quantityRaw,
-    deliveryMethod,
-    ...otherOrderData
-  } = values;
+  if (!hasPermissionToInitiateTransactions(currentUser)) {
```

Comment:
Perhaps we should not add changes to this function either: as handleSubmit does not remove the need to check the rights on CheckoutPage. If we do it there, does this become irrelevant?
Something like this is already on CheckoutPage: https://github.com/sharetribe/web-template/commit/035c124294fe6fe5d123d0aed00cb8bbb3c9749f

Reply:
Updated and moved the check to CheckoutPage!

Reply:
The way this now works on CheckoutPage if the operator restricts the user mid-initiating transaction is that the user sees the error message, and when they refresh, they are redirected. Should the checkout page redirect already as a response to the 403 error from the API?